setup.py

```diff
@@ -139,21 +137,20 @@
 }
 
 
-if 'setuptools' in sys.modules:
-    setup_args.update(setuptools_args)
+# Because of this we do not need a MANIFEST.in
```

Comment:
Then the file can (and should?) be removed?

Reply:
Yes, it is removed in this PR.

Reply:
Oh, I must have missed it in the changed files overview.